recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 3
-  skip: true  # [win32]
-  skip: true  # [win and py<35]
+  number: 0
+  # arrow-cpp is not available for s390x, see arrow-cpp-feedstock for details.
```

Comment:
Should we skip py<37? https://github.com/apache/arrow/blob/c3d031250a7fdcfee5e576833bf6f39097602c30/python/setup.py#L634